src/components/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import Radium from 'radium';
+import React from 'react';
+
+let Link = require('react-router-dom').Link;
+Link = Radium(Link);
+
+class Home extends React.Component {
```

Comment:
Not sure if this is a big deal but `Home` is a rather overloaded/general term. Up to you.

Reply:
Yeah, I wasn't sure what to call it. Any suggestions?

Reply:
Haha :man_shrugging: perhaps we can edit it once we formally get there? I just wasn't sure if you had a specific image in mind of what this view would be.

Reply:
I think we'll keep it as Home for now.